trade-finance-manager-ui/server/controllers/utilisation-reports/record-corrections/initiate-record-correction-request/index.test.ts

```diff
@@ -171,11 +176,29 @@ describe('controllers/utilisation-reports/record-corrections/initiate-record-cor
     );
 
     describe(`when a single checkbox in status ${FEE_RECORD_STATUS.TO_DO} is selected`, () => {
+      const selectedFeeRecordId = '456';
       const requestBody: PremiumPaymentsTableCheckboxSelectionsRequestBody = {
-        'feeRecordIds-456-reportedPaymentsCurrency-GBP-status-TO_DO': 'on',
+        [`feeRecordIds-${selectedFeeRecordId}-reportedPaymentsCurrency-GBP-status-TO_DO`]: 'on',
```

Comment:
This is the checkbox id type so requires all the identifiers, I could revert this change to use the hardcoded value again if this would make it a bit nicer to read though

Reply:
Keep it as is, I think later we will need to relook at the name for these, its too long